<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterconteneur</class>
 <widget class="QDialog" name="ajouterconteneur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Titre">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Ajouter Conteneur :</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdConteneur">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IdConteneur :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeConteneur_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CapaciteMax">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CapaciteMax :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="CapaciteMax_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdConteneur_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TypeConteneur">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TypeConteneur :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Creer">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cr√©er</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Annuler">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
